components/brave_vpn/brave_vpn_service.cc

```diff
@@ -7,6 +7,7 @@
 
 #include "base/json/json_reader.h"
 #include "base/json/json_writer.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
shouldn't we add dependency for `//third_party/abseil-cpp:absl` in `components/brave_vpn/BUILD.gn`?

Reply:
Oops! Indeed, added now. Also, made the most of this force-push to remove this include from chromium_src overrides where the file being overriden already includes it, and remove a temporary include rule from src/DEPs that's not needed anymore.

PTAL and thanks again